examples/gno.land/p/demo/grc/grc20/mock.gno

```diff
@@ -1,3 +0,0 @@
-package grc20
```

Comment:
Does anyone see an interesting mock for GRC20?

Reply:
I thought it was extra, adding it to the todo list🫡